Fruitties/shared/build.gradle.kts

```diff
@@ -82,10 +83,6 @@ android {
     defaultConfig {
         minSdk = 26
     }
-    compileOptions {
```

Comment:
Out of curiosity: why did you have to remove the `compileOptions`?

Reply:
I'll revert this because I did not mean to change it here. I was testing what happens if I remove it because I see `compilerOptions` and `compileOptions` in a few places. I don't have a reason to remove it, but it still compiled for me without it.

`shared` -> kotlin -> compilerOptions
https://github.com/android/kotlin-multiplatform-samples/blob/da242726262ea4563a2ef91f55749ec9decfed4f/Fruitties/shared/build.gradle.kts#L31-L34

`shared` -> android -> compileOptions
https://github.com/android/kotlin-multiplatform-samples/blob/da242726262ea4563a2ef91f55749ec9decfed4f/Fruitties/shared/build.gradle.kts#L85-L88

`androidApp` -> android -> compileOptions
https://github.com/android/kotlin-multiplatform-samples/blob/da242726262ea4563a2ef91f55749ec9decfed4f/Fruitties/androidApp/build.gradle.kts#L45-L48